library/src/main/scala/scala/scalajs/concurrent/JSExecutionContext.scala

```diff
@@ -11,12 +11,14 @@ import scala.concurrent.ExecutionContextExecutor
 object JSExecutionContext {
 
   /** execution context that runs immediately. beware of stack growth! */
+  @deprecated("Not asynchronous. Use JSExecutionContext.queue instead.", "0.6.6")
   val runNow: ExecutionContextExecutor = RunNowExecutionContext
   /** execution context that submits into the JavaScript runtime's
     * task queue */
   val queue: ExecutionContextExecutor = QueueExecutionContext
 
   object Implicits {
+    @deprecated("Not asynchronous. Use JSExecutionContext.Implicits.queue instead.", "0.6.6")
     implicit val runNow: ExecutionContextExecutor = JSExecutionContext.runNow
```

Comment:
Apparently, this line refuses to compile:
```
[info] Compiling 97 Scala sources to /localhome/jenkins/a/workspace/scalajs-task-worker/library/target/scala-2.11/classes...
[error] /localhome/jenkins/a/workspace/scalajs-task-worker/library/src/main/scala/scala/scalajs/concurrent/JSExecutionContext.scala:22: warning: value runNow in object JSExecutionContext is deprecated: Not asynchronous. Use JSExecutionContext.queue instead.
[error]     implicit val runNow: ExecutionContextExecutor = JSExecutionContext.runNow
[error]                                                                        ^
[error] error: No warnings can be incurred under -Xfatal-warnings.
[error] one warning found
[error] one error found
```
It doesn't make any sense to me. It should be allowed to call a deprecated thing from a deprecated thing. But maybe it's because `Implicits.runNow` is a `val`? Anyway, you should be able to work around this by using `RunNowExecutionContext` on the right-hand-side.